jwst/skymatch/skymatch.py

```diff
@@ -456,8 +458,14 @@ def _apply_sky(images, sky_deltas, do_global, do_skysub, show_old):
 #     for i in range(ns):
```

Comment:
Again, why is the commented code left in?

Reply:
Ok, I see it addressed in your original comment, but still, just keep the code around locally until you are satisfied?